docs/install-1.7.md

```diff
@@ -55,8 +53,14 @@ your `minikube start` command with this flag:
 ```console
 minikube start --extra-config=apiserver.Authorization.Mode=RBAC
 ```
+If you are using `hack/local-up-cluster.sh`, ensure the
+`AUTHORIZATION_MODE` environment variable is set as follows:
+
+``console
+AUTHORIZATION_MODE=Node,RBAC hack/local-up-cluster.sh -O
+```
 
-AssumingBy default, `helm init` installs the Tiller pod into the `kube-system`
+Assuming by default, `helm init` installs the Tiller pod into the `kube-system`
```

Comment:
I don't think we need the word "assuming" here.

Reply:
Yes, I agree it. At first, I deal with it as a typos